tests/core/analytics/analyticsreporter.js

```diff
@@ -68,11 +70,11 @@ describe('reporting events', () => {
       expect.anything());
   });
 
-  it('throws error if opted in and ytag missing', () => {
+  it('provide a console error if opted in and ytag missing', () => {
```

Comment:
super nit: missing plural. and actually, can this be "logs a console error" instead?